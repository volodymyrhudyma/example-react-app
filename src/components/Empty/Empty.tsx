import React, { Component } from 'react';
import { Wrapper } from './styled/Empty.styled';
import { WithTranslation } from 'react-i18next';

class Empty extends Component<WithTranslation> {
  render() {
    const { t } = this.props;
    return <Wrapper>{t('resource.empty')}</Wrapper>;
  }
}

export default Empty;
