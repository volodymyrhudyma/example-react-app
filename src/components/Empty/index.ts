import Component from './Empty';
import { withTranslation } from 'react-i18next';

export default withTranslation()(Component);
