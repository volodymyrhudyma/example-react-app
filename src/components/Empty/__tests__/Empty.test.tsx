import { shallow } from 'enzyme';
import React from 'react';
import Empty from '../Empty';
import i18n from 'i18next';

describe('Empty with Enzyme and Jest', () => {
  it('should render correctly', () => {
    const component = shallow(
      <Empty t={jest.fn()} i18n={i18n} tReady={false} />
    );
    expect(component).toMatchSnapshot();
  });
});
