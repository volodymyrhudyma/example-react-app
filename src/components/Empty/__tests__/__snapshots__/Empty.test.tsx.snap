// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty with Enzyme and Jest should render correctly 1`] = `<styled.div />`;
